tools/windows/install-help/cal/TargetMachine.cpp

```diff
@@ -5,8 +5,9 @@
 TargetMachine::TargetMachine()
 : _serverType(0)
 , _machineName(L"")
-, _domain(L"")
+, _joinedDomain(L"")
```

Comment:
because these are `std::wstring` I don't think they need initializers; the default constructor will initialize to empty string.

Reply:
just extra function call overhead